### PR TITLE
fix/expand payload for both attribute instance table

### PR DESCRIPTION
**Pull request by gpoujol**

### CODE REVIEW

File: search-service/src/main/kotlin/db/migration/V0_33__Expand_attribute_instance_payload.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    private lateinit var jdbcTemplate: JdbcTemplate
+
+    override fun migrate(context: Context) {
+        jdbcTemplate = JdbcTemplate(SingleConnectionDataSource(context.connection, true))
+
+        queryAttributeUpdateByPage("attribute_instance", LIMIT, OFFSET)
+        queryAttributeUpdateByPage("attribute_instance_audit", LIMIT, OFFSET)
+    }
+
+    private fun queryAttributeUpdateByPage(tableName: String, limit: Int, offset: Int) {
```
Comment (bobeal) [inline search-service/src/main/kotlin/db/migration/V0_33__Expand_attribute_instance_payload.kt:32]: when does it stop?

Comment (gpoujol) [reply search-service/src/main/kotlin/db/migration/V0_33__Expand_attribute_instance_payload.kt:32]: if `resultSet` retunrs no data, nothing happens then so it stops at this moment. no?

Comment (bobeal) [reply search-service/src/main/kotlin/db/migration/V0_33__Expand_attribute_instance_payload.kt:32]: I just stopped the build in Jenkins that was running for 4 days so no

**Comment by bobeal** [comment]
We can't run it in a migration script since it will take too much (unnecessary) time. So we have first to find a way to run it (once) in a non blocking way.

**Pull request closed without merging**